clj-refactor.el

```diff
@@ -2981,8 +3028,8 @@ See: https://github.com/clojure-emacs/clj-refactor.el/wiki/cljr-find-usages"
         (when (looking-at-p "(\\s-*def")
           (re-search-forward name)
           (paredit-backward))
-        (when (looking-at (format "\\(.+/\\)?\\(%s\\)" name))
-          (replace-match (format "\\1%s" new-name))))
+        (when (re-search-forward (format "\\(.+/\\)?\\(%s\\)" name) search-limit t)
```

Comment:
hm... I think what you mean is #157 as renaming `arg` in this works fine:

```clj
(defn test [arg]
  (println arg arg arg))
 ```

I guess there is no regression here.